zzzz_modular_occulus/code/datums/setup_option/backgrounds/origin.dm

```diff
@@ -1,120 +1,137 @@
+		STAT_VIG = -4,
+		STAT_COG = -2
 	)
 
 
 /datum/category_item/setup_option/background/origin/hmss_destined
+	name = "Crewman"
+	desc = "You're not unfamiliar with working on spaceships. Maybe you've been working for NanoTrasen for some time, or maybe you used to be part of some navy.\
+			 Either way, you know the general ins-and-outs of working on a ship."
 	stat_modifiers = list(
-		STAT_ROB = 7,
-		STAT_TGH = 6,
-		STAT_BIO = 0,
-		STAT_MEC = 8,
-		STAT_VIG = 2,
-		STAT_COG = -8
+		STAT_ROB = 1,
+		STAT_TGH = 1,
+		STAT_BIO = -2,
+		STAT_MEC = 2,
+		STAT_COG = -4
 	)
 
 
 /datum/category_item/setup_option/background/origin/first_expeditionary_fleet
+	name = "Activist"
+	desc = "You have a strong set of beliefs, and stick to them. You're not afraid to clash with authority when push comes to shove. \
+			Maybe you're even on the CEV Northern Light because you're running from the law. Either way, you're still carrying out your campaign for justice."
+
 	stat_modifiers = list(
-		STAT_ROB = 5,
-		STAT_TGH = 0,
-		STAT_BIO = 5,
-		STAT_MEC = -5,
-		STAT_VIG = 7,
-		STAT_COG = 6
+		STAT_COG = 4,
+		STAT_ROB = 1,
+		STAT_BIO = 1,
+		STAT_MEC = -3,
+		STAT_BIO = -2,
+		STAT_VIG = 1
 	)
 
 /datum/category_item/setup_option/background/origin/nss_forecaster
+	name = "Nomad"
+	desc = "You used to roam the stars. Maybe you still do. Why would you miss your home when your home is all around you? There's so much to see, and you are ready for all of it! \
+			Your lack of settling down means that you don't have intimate knowledge of any particular subject, but you've always been able to get around that with your general knowledge and wit.\
+			So far, at least."
 	stat_modifiers = list(
-		STAT_ROB = 4,
-		STAT_TGH = 4,
-		STAT_BIO = -10,
-		STAT_MEC = 4,
-		STAT_VIG = 10,
-		STAT_COG = 6
+		STAT_BIO = -2,
+		STAT_COG = -4,
+		STAT_MEC = 2,
+		STAT_VIG = 2
 	)
 
 /datum/category_item/setup_option/background/origin/sich_prime
 	desc = "Miscreant. You're used to having to do what it takes to survive. Pulling out bullets, picking pockets, and  picking locks... it's all the same to you with your nimble fingers."
 	stat_modifiers = list(
 		STAT_ROB = -2,
 		STAT_TGH = -3,
-		STAT_BIO = 6,
-		STAT_MEC = 6,
-		STAT_VIG = 6,
-		STAT_COG = 4
-	)
\ No newline at end of file
+		STAT_BIO = 3,
+		STAT_MEC = 2
+	)
+
+/datum/category_item/setup_option/background/origin/medic
+	name = "Medic"
+	desc = "Be it by choice or cosmic design, most of your life has been spent tending to living things: plants, animals, or people. You have a strong understanding of what makes us all tick."
+	stat_modifiers = list(
+		STAT_MEC = -5,
+		STAT_BIO = 5
+	)
+/*
+/datum/category_item/setup_option/background/origin/medic
+	name = "Medic"
+	desc = "Be it by choice or cosmic design, most of your life has been spent tending to living things: plants, animals, or people. You have a strong understanding of what makes us all tick."
+	stat_modifiers = list(
+		STAT_MEC -5,
+		STAT_BIO = 5
+	)
+*/
```

Comment:
dupe of code immediately above, looks like

Reply:
It's commented out. Bad indentation

Reply:
But why is this here?